definitions/v3/xthor-api.yml

```diff
@@ -0,0 +1,179 @@
+
+    # TV Series / Series TV
+    - { id: 104, cat: TV/Other, desc: "Series/BluRay" }
+    - { id: 13, cat: TV, desc: "Series/Pack VF" }
+    - { id: 15, cat: TV/HD, desc: "Series/HD VF" }
+    - { id: 14, cat: TV/SD, desc: "Series/SD VF" }
+    - { id: 98, cat: TV/Other, desc: "Series/Pack VOSTFR" }
+    - { id: 17, cat: TV/HD, desc: "Series/HD VOSTFR" }
+    - { id: 16, cat: TV/SD, desc: "Series/SD VOSTFR" }
+    - { id: 101, cat: TV/Anime, desc: "Series/Packs Anime" }
+    - { id: 32, cat: TV/Anime, desc: "Series/Animes" }
+    - { id: 110, cat: TV/Anime, desc: "Series/Anime VOSTFR" }
+    - { id: 123, cat: TV/Other, desc: "Series/Animation" }
+    - { id: 109, cat: TV/Documentary, desc: "Series/DOC" }
+    - { id: 34, cat: TV/Sport, desc: "Series/Sport" }
+    - { id: 30, cat: TV/Other, desc: "Series/Emission TV" }
+
+    # Porn / XxX
+    - { id: 36, cat: XXX, desc: "MISC/XxX/Films" }
+    - { id: 105, cat: XXX, desc: "MISC/XxX/Séries" }
+    - { id: 114, cat: XXX, desc: "MISC/XxX/Lesbiennes" }
+    - { id: 115, cat: XXX, desc: "MISC/XxX/Gays" }
+    - { id: 113, cat: XXX, desc: "MISC/XxX/Hentai" }
+    - { id: 120, cat: XXX, desc: "MISC/XxX/Magazines" }
+
+    # Books / Livres
+    - { id: 24, cat: Books/EBook, desc: "Livres/Romans" }
+    - { id: 124, cat: Audio/Audiobook, desc: "Livres/Audio Books" }
+    - { id: 96, cat: Books/Mags, desc: "Livres/Magazines" }
+    - { id: 99, cat: Books/Other, desc: "Livres/Bandes dessinées" }
+    - { id: 116, cat: Books/EBook, desc: "Livres/Romans Jeunesse" }
+    - { id: 102, cat: Books/Comics, desc: "Livres/Comics" }
+    - { id: 103, cat: Books/Other, desc: "Livres/Mangas" }
+
+    # Softwares / Logiciels
+    - { id: 25, cat: PC/Games, desc: "Logiciels/Jeux PC" }
+    - { id: 27, cat: Console/PS3, desc: "Logiciels/Playstation" }
+    - { id: 111, cat: PC/Mac, desc: "Logiciels/Jeux MAC" }
+    - { id: 26, cat: Console/XBox 360, desc: "Logiciels/XboX" }
+    - { id: 112, cat: PC, desc: "Logiciels/Jeux Linux" }
+    - { id: 28, cat: Console/Wii, desc: "Logiciels/Nintendo" }
+    - { id: 29, cat: Console/NDS, desc: "Logiciels/NDS" }
+    - { id: 117, cat: PC, desc: "Logiciels/ROM" }
+    - { id: 21, cat: PC, desc: "Logiciels/Applis PC" }
+    - { id: 22, cat: PC/Mac, desc: "Logiciels/Applis Mac" }
+    - { id: 23, cat: PC/Mobile-Android, desc: "Logiciels/Smartphone" }
+
+  modes:
+    search: [ q ]
+    movie-search: [ q, tmdbid ]
+    tv-search: [ q, season, ep ]
+    book-search: [ q ]
+
+settings:
+  - name: passkey
+    type: password
+    label: Passkey
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: multilanguage
+    type: text
+    label: Replace MULTI by
+    default: MULTI
+  - name: subfrench
+    type: text
+    label: Replace "VOSTFR/SUBFRENCH" by
+    default:
+
+search:
+  paths:
+    # https://api.xthor.tk/?passkey=<redacted>&search=&category=&freeleech=&tmdbid=&size=&accent=&price=&page=
+    - path: "https://api.xthor.tk/"
```

Comment:
this shouldn't be the site hardcoded - it should use the URL from the config ideally.

not sure the past way to handle prepending `api.` to the config url

Reply:
Yes, i know that's not ideal, but if you can tell me how to do it properly with a subdomain by appending it to the URL from config

Reply:
defer to Qstick

Reply:
could just use `https://api.xthor.tk/` in links section. Otherwise it should be fine like this for now as long as they are not often changing domains